internal/engine/wazevo/e2e_test.go

```diff
@@ -25,6 +27,16 @@ const (
 	f64 = wasm.ValueTypeF64
 )
 
+func TestSpectestV1(t *testing.T) {
+	config := wazero.NewRuntimeConfigCompiler().WithCoreFeatures(api.CoreFeaturesV1)
+	// Configure the new optimizing backend!
+	configureWazevo(config)
+
+	// TODO: adds incrementally one by one as we support more test cases. And eventually remove this
+	// 	and migrate to spectest/v2/spec_test.go by the time when closing https://github.com/tetratelabs/wazero/issues/1496
+	spectest.RunJson(t, v1.Testcases, "binary.json", context.Background(), config)
```

Comment:
@evacchi this is where you can add more tests from the spectests, and find the missing features/bugs

Reply:
🫡